server/middleware/setUpSentry.ts

```diff
@@ -27,6 +27,23 @@ function setUpSentryRequestHandler(app: express.Express): void {
         // enable Express.js middleware tracing
         new Sentry.Integrations.Express({ app }),
       ],
+      tracesSampler: samplingContext => {
+        const transactionName = samplingContext?.transactionContext?.name
+        if (
+          transactionName?.includes('ping') ||
+          transactionName?.includes('health') ||
+          transactionName?.includes('assets')
+        ) {
+          return 0
+        }
+
+        if (config.environment === 'prod') {
+          return 1
+        }
```

Comment:
I wonder if it's worth reframing the preamble in the commit message, given the sampling doesn't happen in prod, which presumably would be the environment with the most traffic (the unordered list halfway through is a qualification of the benefit mentioned up until then)?

Also, in the commit message, I don't think `[1]` does anything on its own. I think it needs to be connected to a square bracketed bit of text. So `[text](link)` would become `[text][ref]` and `[ref]: link`. Not that GitHub parses that in commit messages anyway

Reply:
Have reworded the commit message here.

I'm not sure if `[1]` is a markdown thing or something else, but it's more of a convention people use for footnoting things in markdown documents I've seen!

Reply:
Ah, I thought you were trying to do the linking by reference thing